src/main/java/frc/robot/RobotContainer.java

```diff
@@ -109,7 +111,7 @@ public void putShuffleboardCommands() {
         if (Constants.FEEDER_DEBUG){
             SmartShuffleboard.putCommand("Feeder", "Feed", new StartFeeder(feeder));
         }
-
+        SmartShuffleboard.putCommand("Feeder", "StartFeeder", new FeederColorMatcher(feeder, colorSensor));
```

Comment:
Lets move this into the "if (Constants.FEEDER_DEBUG)" section like the other command.